modules/account/src/main/kotlin/edu/stanford/spezi/module/account/manager/UserSessionManager.kt

```diff
@@ -79,11 +82,24 @@ internal class UserSessionManagerImpl @Inject constructor(
         )
     }
 
+    override suspend fun forceRefresh() {
+        firebaseAuth.currentUser?.getIdToken(true)?.await()
+    }
+
+    @Suppress("UnusedPrivateMember")
```

Comment:
Is there any plan to add the consent again?

Other question: What should happen with other related components: ConsentScreen, related routes and navigation events and the pdf creation service? All these are only used in tests now. Should we at least remove the consent screen from the nav graph in main activity? If so we should adjust the app navigation tests as well

Reply:
consent screen is also part of the spezi framework, so it will still be there, even if the app itself does not currently use it. the code for this should therefore remain in place for the time being. removing the consent screen from nav graph is a good idea.